nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,89 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const meals = require("./data/meals");
+const reservations = require("./data/reservations");
+const reviews = require("./data/reviews");
+
+//Adding a function to get the reviews for every meal
+function getReviews() {
+  meals.map((meal) => {
+    let matchedReviews = [];
```

Comment:
const, you are using map-like a foreach. check here for more info :)
https://codeburst.io/javascript-map-vs-foreach-f38111822c0f